fastlane/Fastfile

```diff
@@ -8,12 +8,36 @@ fastlane_require 'buildkit'
 
 UI.user_error!('Please run fastlane via `bundle exec`') unless FastlaneCore::Helper.bundler?
 
+PROJECT_ROOT_FOLDER = File.dirname(File.expand_path(__dir__))
+VERSION_FILE_PATH = File.join(PROJECT_ROOT_FOLDER, 'config', 'Version.Public.xcconfig')
+OUTPUT_DIRECTORY_PATH = File.join(PROJECT_ROOT_FOLDER, 'build', 'results')
+APP_STORE_BUNDLE_IDENTIFIER = 'com.codality.NotationalFlow'
+DEFAULT_BRANCH = 'trunk'
```

Comment:
All these consts at the top were previously later on in the file I grouped them for clarity and because of the new `import`s that require them to be already defined.